ts/screens/authentication/SpidInformationRequestScreen.tsx

```diff
@@ -28,27 +28,27 @@ type OwnProps = {
 };
 type Props = ReduxMappedProps & OwnProps;
 type State = {
-  isTosModalVisible: boolean;
+  isHelpVisible: boolean;
 };
 /**
  * A screen where the user can insert an email to receive information about SPID.
  */
 class SpidInformationRequestScreen extends React.Component<Props, State> {
   public state: State = {
-    isTosModalVisible: false
+    isHelpVisible: false
   };
 
   private goBack() {
     this.props.navigation.goBack();
   }
 
-  private showModal() {
-    this.setState({ isTosModalVisible: true });
-  }
+  private showHelp = () => {
```

Comment:
if this code is supposed to be the same through different components, 
maybe it's worth to make some mixin (HOC) so you can use something like:

`export withHelp(SpidInformationRequestScreen, myHelpComponent);`

this has the nice side effect of making `myHelpComponent `reusable through different parent components
(good for decoupling)

see https://reactjs.org/docs/higher-order-components.html

Reply:
Sweet! I did not consider this option, will definitely implement it

Reply:
I think we haven't others HOC in the project but i definitely agree in this case a HOC is the right approach.